build.gradle

```diff
@@ -0,0 +1,28 @@
+plugins {
+	id 'java'
+	id 'org.springframework.boot' version '3.4.1'
+	id 'io.spring.dependency-management' version '1.1.7'
+}
+
+group = 'com.delo'
+version = '0.0.1-SNAPSHOT'
+
+java {
+	toolchain {
+		languageVersion = JavaLanguageVersion.of(23)
+	}
+}
+
+repositories {
+	mavenCentral()
```

Comment:
where is your local and remote repository located ? can you point to location/add screenshot ?

Reply:
**Local Repository**
<img width="1081" alt="local" src="https://github.com/user-attachments/assets/d2754bea-1bd4-4ff7-acea-c51f324526c3" />

Reply:
dependencies {
	implementation 'org.springframework.boot:spring-boot-starter-web'
	testImplementation 'org.springframework.boot:spring-boot-starter-test'
	testRuntimeOnly 'org.junit.platform:junit-platform-launcher'
}

Reply:
org.springframework.boot:spring-boot-starter-web where this is stored in local and what is url /site from where it is downloaded and stored in local

Reply:
<img width="1440" alt="repo" src="https://github.com/user-attachments/assets/5608ba66-a15b-4f17-92b3-6018d7d9be7c" />